desc/plotting.py

```diff
@@ -946,9 +971,156 @@ def plot_fsa(
         from desc.plotting import plot_fsa
         fig, ax = plot_fsa(eq, "B_theta")
 
+    """
+    return _plot_surface_averages(
+        eq=eq,
+        name=name,
+        use_sqrt_g=True,
+        log=log,
+        rho=rho,
+        M=M,
+        N=N,
+        norm_F=norm_F,
+        ax=ax,
+        return_data=return_data,
+        **kwargs,
+    )
+
+
+def plot_tsa(
```

Comment:
What is the use case for this?

Also, from a naming perspective, a theta surface to me sounds like theta=const, rho,zeta vary. So in keeping with the idea of a `fsa` being a function of rho with theta/zeta averaged out, a `tsa` sounds like it should be a function of theta, with rho/zeta averaged out?

Reply:
Sometimes the surface average of interest of a quantity does not include the sqrt(g) factor. For example, B_theta is a quantity where the surface average without the sqrt(g) factor is the toroidal current. I don't know what a surface average of B_theta with the sqrt(g) factor represents.

The plotted example quantity for plot_fsa given in the docs is B_theta. It's likely that the intent of plotting that quantity should be done by function like plot_tsa instead, which does not use sqrt(g), so I added the plot_tsa function.

I agree that 'theta surface average' is poorly named. I called it that because the paper I linked made it seem like a known standard convention.

Reply:
OK. I'd vote for removing `plot_tsa` and just adding an option to not include `sqrt(g)` in `plot_fsa`
As for renaming `plot_fsa` to `plot_surface_average` I'd lean towards not, just to keep a consistent API but I could go either way. If we do change it, we should keep `plot_fsa` as a wrapper to `plot_surface_averages` for now and have it throw a deprecation warning until we remove it fully in a future version (and in general we should start trying to make use of deprecation warnings rather than just suddenly changing API stuff where possible.)